mdk/build.gradle

```diff
@@ -71,7 +71,7 @@ processResources
         expand 'version':project.version, 'mcversion':project.minecraft.version
     }
         
-    // copy everything else, thats not the mcmod.info
+    // copy everything else except the mcmod.info
```

Comment:
Thanks a lot for fixing a minor issue that has existed since start of the use of gradle